packages/@n8n/api-types/src/dto/credentials/create-credential.dto.ts

```diff
@@ -3,7 +3,7 @@ import { Z } from 'zod-class';
 
 export class CreateCredentialDto extends Z.class({
 	name: z.string().min(1).max(128),
-	type: z.string().min(1).max(32),
+	type: z.string().min(1).max(64),
```

Comment:
Since the table schema allows 128 characters here

```suggestion
	type: z.string().min(1).max(128),
```